src/views/HomeView.vue

```diff
@@ -51,41 +51,46 @@ onMounted(() => {
 
 <template>
   <div class="grid-container">
-    <div class="header">
+    <header class="header">
       <HeaderBar
         :all-sports="allSports"
         :name="allSports.name"
       />
-    </div>
-    <div class="sidebar">
+    </header>
+    <aside class="sidebar">
       <SideBar />
-    </div>
-    <div class="body">
+    </aside>
+    <body class="body">
       body
-    </div>
+    </body>
```

Comment:
This is better. But `body` is a reserved term, so maybe something better there.

Also, leave a line between equal indented html blocks, it makes it more readable.

See if there's places in the rest of the code where you can apply semantic HTML, not just in this section.